### PR TITLE
Update AWS Native blog title

### PR DESCRIPTION
**Pull request by infin8x**

### CODE REVIEW

**Review by nimbinatus — APPROVED** [review]
LGTM

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
-title: "Announcing the Pulumi AWS Native Provider"
+title: "Announcing the Pulumi AWS Native Provider, powered by the AWS Cloud Control API"
```
Comment (isaac-pulumi) [inline themes/default/content/blog/announcing-aws-native/index.md:2]:
```suggestion
title: "Announcing the Pulumi AWS Native Provider, Powered by the AWS Cloud Control API"
```

Comment (isaac-pulumi) [reply themes/default/content/blog/announcing-aws-native/index.md:2]: suggest applying title case

**Review by isaac-pulumi — CHANGES_REQUESTED** [review]

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 1 above)
Comment (infin8x) [reply themes/default/content/blog/announcing-aws-native/index.md:2]: We should write down a consistent policy here at some point. I'm personally not a fan of title case, it feels very shout-y.  But, I'll take it here for now.

**Review by isaac-pulumi — APPROVED** [review]

**Pull request merged**